scripts/functions/utility_system

```diff
@@ -146,12 +146,12 @@ __list_remote_all()
   while
     __rvm_db "rvm_remote_server_url${_iterator:-}" rvm_remote_server_url
   do
-    __rvm_include_travis_binaries || continue
+    if __rvm_include_travis_binaries; then
```

Comment:
https://github.com/rvm/rvm/blob/master/FORMATTING.md#sparse-form-for-long-statements--more-commands

Reply:
Sorry, I just used the style from the `__rvm_remote_server_path` function directly above in the same file, which used `if ; then`.

I pushed a new version that follows the style guide and also simplified the commit message.